src/main/java/org/webshar/hrms/request/organization/OrganizationCreateRequest.java

```diff
@@ -0,0 +1,36 @@
+package org.webshar.hrms.request.organization;
+
+import javax.validation.constraints.AssertTrue;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.webshar.hrms.constants.ErrorMessageConstants;
+
+public class OrganizationCreateRequest
+{
+
+  @NotNull(message = ErrorMessageConstants.ORGANIZATION_NAME_NOT_NULL)
+  @NotBlank(message = ErrorMessageConstants.ORGANIZATION_NAME_NOT_BLANK)
+  @Getter
+  @Setter
+  private String name;
+
+  @Getter
+  @Setter
+  private Boolean isActive;
+
+  @AssertTrue(message = ErrorMessageConstants.ORGANIZATION_INVALID_STATAUS)
+  private boolean isValidStatus()
+  {
+    if (isActive == null)
+    {
+      return true;
+    }
+    else if (isActive != true && isActive != false)
+    {
+      return false;
+    }
+    return true;
```

Comment:
This logic is not making sense. Possible values are true, false, null and As per code, it is always isValidStatus returning true in all cases.